tests/Integration/IntegrationTest.php

```diff
@@ -57,4 +58,36 @@ public function testAddAndRetrieveAsRange(): void
 
         $this->assertEquals($expectedRange, $range);
     }
+
+    public function testAddAndRetrieveAsMRangeAndMultipleFilters(): void
+    {
+        $from = new DateTimeImmutable('2019-11-06 20:34:17.103000');
+        $to = new DateTimeImmutable('2019-11-06 20:34:17.107000');
+
+        $this->sut->create(
+            'temperature:3:11',
+            6000,
+            [new Label('sensor_id', '2'), new Label('area_id', '32')]
+        );
+        $this->sut->add(new Sample('temperature:3:11', 30, $from));
+        $this->sut->add(new Sample('temperature:3:11', 42, $to));
+
+        $filter = new Filter('sensor_id', '2');
+        $filter->add('area_id', Filter::OP_EQUALS, '32');
+
+        $range = $this->sut->multiRange(
+            $filter,
+            null,
+            null,
+            null,
+            null
```

Comment:
Those are not needed, apart from the first, all the other parameters have default value = null.

Reply:
I wanted to be explicit about it. But I'll remove it.